### ISSUE TITLE
No Response to Experimental API requests

### ISSUE THREAD

**Issue by nickfausnight-zz** [hubski/hubski-dot-com#2]
Not sure if this is a bug report or a feature request since the api is experimental.

Sending a `GET` request to `https://api.hubski.com/publication/336207/tree` and `https://api.hubski.com/publication/336207/` both return no response.

Expected: a JSON response representing a publication from both endpoints, and the publication's children from the tree endpoint.

Actual: No response.

Am I sending my requests to the right endpoint?

**Comment by rob05c** [comment]
Yes, the API is down. The issue is that the hubski.com HTTPS certificate is now served on the front-end server, while the API is on the back-end server, which doesn't have the cert.

We're working on getting the cert to the back-end server. Thanks for the bug report, though, it needed one.
